package.json

```diff
@@ -812,8 +812,8 @@
         "cleanReadme": "gulp cleanReadme",
         "compile": "tsc -watch",
         "package": "vsce package --githubBranch main",
-        "lint": "eslint -c .eslintrc.js --ext .ts .",
-        "lint-fix": "eslint -c .eslintrc.js --ext .ts . --fix",
+        "lint": "eslint --ext .ts .",
+        "lint-fix": "eslint --ext .ts . --fix",
```

Comment:
For ESLint to use the `.eslintrc.js` in `test/` I had to remove the config file option here